WebDevStudios/wd_s
# Update webdevstudios/php-coding-standards to ~1.3.0

No ticket #.

### DESCRIPTION

Updates WDS PHP coding standards to ~1.3.0
This will alleviate the `Short array syntax is not allowed` "error" when running PHPCS (ref https://github.com/WebDevStudios/php-coding-standards/issues/40).

### OTHER

- [x ] Does this issue pass all the linting? (PHPCS, ESLint, SassLint)

### STEPS TO VERIFY

Prior to this PR, using WDS PHP coding standards to ~1.2.1,
1. Add an array to some PHP code using short array syntax
1. Run `composer install`
1. Run `composer run lint:php`
Observe PHPCS `Short array syntax is not allowed` errors (example):
```
09:02 $ composer run lint:php
> composer run lint:php-compat && composer run lint:phpcs
> @php ./vendor/squizlabs/php_codesniffer/bin/phpcs -p . --standard=PHPCompatibility --extensions=php --runtime-set testVersion 7.4 --ignore='.github/*,vendor/*' --warning-severity=8 -d memory_limit=-1
..................................... 37 / 37 (100%)


Time: 7.25 secs; Memory: 12MB

> @php ./vendor/squizlabs/php_codesniffer/bin/phpcs -d memory_limit=-1 --standard=./.phpcs.xml.dist --extensions=php -n --colors **/*.php

FILE: ...ites/wd_s/app/public/wp-content/themes/wd_s/inc/scripts.php
----------------------------------------------------------------------
FOUND 6 ERRORS AFFECTING 6 LINES
----------------------------------------------------------------------
  41 | ERROR | [x] Short array syntax is not allowed
  47 | ERROR | [x] Short array syntax is not allowed
  54 | ERROR | [x] Short array syntax is not allowed
  77 | ERROR | [x] Short array syntax is not allowed
  85 | ERROR | [x] Short array syntax is not allowed
 112 | ERROR | [x] Short array syntax is not allowed
----------------------------------------------------------------------
PHPCBF CAN FIX THE 6 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------

Time: 1.81 secs; Memory: 14MB

Script @php ./vendor/squizlabs/php_codesniffer/bin/phpcs -d memory_limit=-1 --standard=./.phpcs.xml.dist --extensions=php -n --colors **/*.php handling the lint:phpcs event returned with error code 2
Script composer run lint:php-compat && composer run lint:phpcs handling the lint:php event returned with error code 2
```

After this PR, using WDS PHP coding standards to ~1.3.0,
1. Add an array to some PHP code using short array syntax
1. Run `composer install`
1. Run `composer run lint:php`
Observe PHPCS `Short array syntax is not allowed` errors are gone (example):
```
09:08 $ composer run lint:php
> composer run lint:php-compat && composer run lint:phpcs
> @php ./vendor/squizlabs/php_codesniffer/bin/phpcs -p . --standard=PHPCompatibility --extensions=php --runtime-set testVersion 7.4 --ignore='.github/*,vendor/*' --warning-severity=8 -d memory_limit=-1
..................................... 37 / 37 (100%)


Time: 3.16 secs; Memory: 12MB

> @php ./vendor/squizlabs/php_codesniffer/bin/phpcs -d memory_limit=-1 --standard=./.phpcs.xml.dist --extensions=php -n --colors **/*.php
```

### DOCUMENTATION

I do not believe the documentation will need to be updated.
